cpp/template/CMakeLists.txt

```diff
@@ -36,3 +36,8 @@ include(cmake/thirdparty/get_raft.cmake)
 # -------------- compile tasks ----------------- #
 add_executable(TEST_RAFT src/test_vector_search.cu)
```

Comment:
Can we change this to `CAGRA_EXAMPLE`? It's no longer really a RAFT test and after consolidating with the ivf-flat example, it'll be consistent.